File: Client/php/GetData/GetDataRequestBuilder.php
<?php

namespace ApiSdk\GetData;

use ApiSdk\GetData\Sequence\SequenceRequestBuilder;
use ApiSdk\GetData\Track\TrackRequestBuilder;
use Microsoft\Kiota\Abstractions\BaseRequestBuilder;
use Microsoft\Kiota\Abstractions\RequestAdapter;

/**
 * Builds and executes requests for operations under /getData
*/
class GetDataRequestBuilder extends BaseRequestBuilder 
{
    /**
     * The sequence property
    */
    public function sequence(): SequenceRequestBuilder {
        return new SequenceRequestBuilder($this->pathParameters, $this->requestAdapter);
    }
    
    /**
     * The track property
    */
    public function track(): TrackRequestBuilder {
        return new TrackRequestBuilder($this->pathParameters, $this->requestAdapter);
    }
    
    /**
     * Instantiates a new GetDataRequestBuilder and sets the default values.
     * @param array<string, mixed>|string $pathParametersOrRawUrl Path parameters for the request or a String representing the raw URL.
     * @param RequestAdapter $requestAdapter The request adapter to use to execute the requests.
    */
    public function __construct($pathParametersOrRawUrl, RequestAdapter $requestAdapter) {
        parent::__construct($requestAdapter, [], '{+baseurl}/getData');
        if (is_array($pathParametersOrRawUrl)) {
            $this->pathParameters = $pathParametersOrRawUrl;
        } else {
            $this->pathParameters = ['request-raw-url' => $pathParametersOrRawUrl];
        }
    }

}

File: Client/go/getdata/get_data_request_builder.go
package getdata

import (
    i2ae4187f7daee263371cb1c977df639813ab50ffa529013b7437480d1ec0158f "github.com/microsoft/kiota-abstractions-go"
)

// GetDataRequestBuilder builds and executes requests for operations under \getData
type GetDataRequestBuilder struct {
    i2ae4187f7daee263371cb1c977df639813ab50ffa529013b7437480d1ec0158f.BaseRequestBuilder
}
// NewGetDataRequestBuilderInternal instantiates a new GetDataRequestBuilder and sets the default values.
func NewGetDataRequestBuilderInternal(pathParameters map[string]string, requestAdapter i2ae4187f7daee263371cb1c977df639813ab50ffa529013b7437480d1ec0158f.RequestAdapter)(*GetDataRequestBuilder) {
    m := &GetDataRequestBuilder{
        BaseRequestBuilder: *i2ae4187f7daee263371cb1c977df639813ab50ffa529013b7437480d1ec0158f.NewBaseRequestBuilder(requestAdapter, "{+baseurl}/getData", pathParameters),
    }
    return m
}
// NewGetDataRequestBuilder instantiates a new GetDataRequestBuilder and sets the default values.
func NewGetDataRequestBuilder(rawUrl string, requestAdapter i2ae4187f7daee263371cb1c977df639813ab50ffa529013b7437480d1ec0158f.RequestAdapter)(*GetDataRequestBuilder) {
    urlParams := make(map[string]string)
    urlParams["request-raw-url"] = rawUrl
    return NewGetDataRequestBuilderInternal(urlParams, requestAdapter)
}
// Sequence the sequence property
// returns a *SequenceRequestBuilder when successful
func (m *GetDataRequestBuilder) Sequence()(*SequenceRequestBuilder) {
    return NewSequenceRequestBuilderInternal(m.BaseRequestBuilder.PathParameters, m.BaseRequestBuilder.RequestAdapter)
}
// Track the track property
// returns a *TrackRequestBuilder when successful
func (m *GetDataRequestBuilder) Track()(*TrackRequestBuilder) {
    return NewTrackRequestBuilderInternal(m.BaseRequestBuilder.PathParameters, m.BaseRequestBuilder.RequestAdapter)
}
